.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
+        node_version: [12, 13]
+    name: CI on NodeJS v${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - run: git config --global core.autocrlf false
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node_version }}
+      - run: npm run lint
+      - run: npm run checkstyle
+      - run: npm run test:coveralls
```

Comment:
`npm` instead of `yarn`, as I based this on the default `actions/setup-node@v1` and `npm` is there by default, but probably `yarn` not.

Reply:
We use Yarn and checkin lockfiles so we should be using Yarn in CI for a test environment that matches what we see locally. If newer than pinned dependencies cause problems, we find out via Greenkeeper PRs anyway and can deal with it without it blocking other work.

Reply:
I'll follow https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-nodejs-with-github-actions#starting-with-the-nodejs-workflow-template